ompi/mca/coll/han/coll_han_subcomms.c

```diff
@@ -276,10 +276,10 @@ int mca_coll_han_comm_create(struct ompi_communicator_t *comm,
     OBJ_CONSTRUCT(&comm_info, opal_info_t);
 
     /*
-     * Upgrade sm module priority to set up low_comms[0] with sm module
+     * Upgrade tuned module priority to set up low_comms[0] with tuned module
      * This sub-communicator contains the ranks that share my node.
      */
-    opal_info_set(&comm_info, "ompi_comm_coll_preference", "tuned,^han");
+    opal_info_set(&comm_info, "ompi_comm_coll_preference", "tuned,^han,^sm");
```

Comment:
Not sure that this is necessary, but anyway if memory serves right, it has to be `tuned,^han,sm` (only one `^`)

Reply:
@gkatev @wenduwan That is correct -- there should only be a single `^`